db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
```

Comment:
Consider using `null=True` for the `description` field in the `Race` model if you want to store `NULL` in the database when no description is provided. The `blank=True` option only affects form validation, not the database schema.